src/app/components/GalleryGrid/GalleryGridItem.tsx

```diff
@@ -0,0 +1,39 @@
+import Image, { type StaticImageData } from "next/image";
+import { IconLoupe } from "@/app/icons/IconLoupe";
+
+interface GalleryGridItemProps {
+  imageUrl: string | StaticImageData;
+  altText: string;
+  priority: boolean;
+  onClick: () => void;
+}
+
+export const GalleryGridItem = ({
+  imageUrl,
+  altText,
+  priority,
+  onClick,
+}: GalleryGridItemProps) => {
+  return (
+    <li
+      className="group relative aspect-square w-full cursor-pointer overflow-hidden"
+      onClick={onClick}
+    >
+      <Image
+        src={imageUrl}
+        alt={altText}
+        fill
+        sizes="25vw"
```

Comment:
![image](https://github.com/user-attachments/assets/3113c4d8-6c08-476a-8b10-3dfef9058767)
![image](https://github.com/user-attachments/assets/5a65b23b-b9bc-4e4d-beca-2bde3392c325)
You are using relative units, which makes sense in the case of dividing a container into an even grid. However, the reference point for the vw is the Container component, which not only has breakpoint widths assigned, but also paddings that have different values than those in the gallery. Therefore, the sizes of the thumbnail images are different from the design in Figma. 

If you want to stick as close to the design as possible, it is worth improving the display of these elements.